test/tools/config_load_check/config_load_check.cc

```diff
@@ -19,6 +22,11 @@ int main(int argc, char* argv[]) {
     return EXIT_FAILURE;
   }
   try {
+
+    Envoy::Event::Libevent::Global::initialize();
+    static auto* lock = new Envoy::Thread::MutexBasicLockable();
```

Comment:
Can we just do a local definition and pass this into `initialize`, since we're basically at the outer scope of the entire execution, or do we need to ensure this lives forever?

Reply:
A local definition is sufficient. It was a copy/pasta error. Fixed in the latest pr.